apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -116,6 +116,8 @@ type TempoStackSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Observability"
 	Observability ObservabilitySpec `json:"observability,omitempty"`
+
+	ReceiversTLS ReceiversTLSSpec `json:"receiversTLS,omitempty"`
```

Comment:
I suggest to put this configuration under `.spec.template.distributor.tls` (TLS config of the distributor component) instead of a new top-level `.spec.receiversTLS`.